assets/js/googlesitekit/datastore/user/key-metrics.test.js

```diff
@@ -631,30 +631,6 @@ describe( 'core/user key metrics', () => {
 				).toBe( false );
 			} );
 
-			it( 'should return false if a module that the widget depends on is not connected', () => {
```

Comment:
Can we change this so it tests the inverse which is now the case? It would be good to codify that behaviour in a test, and since it's already here should be straightforward to change to suit the new behaviour.

Reply:
@tofumatt this has been addressed in https://github.com/google/site-kit-wp/pull/7604/commits/8eb9d59f1c7d4caa4cde9eea0b92fa624908b66b